sdk/eventhub/azure-eventhubs/README.rst

```diff
@@ -1,33 +1,229 @@
+
+- `Send event data`_
+- `Receive event data`_
+- `Async send event data`_
+- `Async receive event data`_
+
+.. _`Send event data`:
+
+Send event data
+---------------
+
+Sends an event data and blocks until acknowledgement is received or operation times out.
+
+.. code:: python
+
+    client = EventHubClient.from_connection_string(connection_str)
+    sender = client.add_sender(partition="0")
+     try:
+        client.run()
+        event_data = EventData(b"A single event")
+        sender.send(event_data)
+    except:
+        raise
+    finally:
+        client.stop()
+
+.. _`Receive event data`:
+
+Receive event data
+------------------
+
+Receive events from the EventHub.
+
+.. code:: python
+
+    client = EventHubClient.from_connection_string(connection_str)
+    receiver = client.add_receiver(consumer_group="$default", partition="0", offset=Offset('@latest'))
+     try:
+        client.run()
+        logger = logging.getLogger("azure.eventhub")
+        received = receiver.receive(timeout=5, max_batch_size=100)
+        for event_data in received:
+            logger.info("Message received:{}".format(event_data.body_as_str()))
+    except:
+        raise
+    finally:
+        client.stop()
+
+.. _`Async send event data`:
+
+Async send event data
+---------------------
+
+Sends an event data and asynchronously waits until acknowledgement is received or operation times out.
+
+.. code:: python
+
+    client = EventHubClientAsync.from_connection_string(connection_str)
+    sender = client.add_async_sender(partition="0")
+    try:
+        await client.run_async()
+        event_data = EventData(b"A single event")
+        await sender.send(event_data)
+    except:
+        raise
+    finally:
+        await client.stop_async()
+
+.. _`Async receive event data`:
+
+Async receive event data
+------------------------
+
+Receive events asynchronously from the EventHub.
+
+.. code:: python
+
+    client = EventHubClientAsync.from_connection_string(connection_str)
+    receiver = client.add_async_receiver(consumer_group="$default", partition="0", offset=Offset('@latest'))
+    try:
+        await client.run_async()
+        logger = logging.getLogger("azure.eventhub")
+        received = await receiver.receive(timeout=5)
+        for event_data in received:
+            logger.info("Message received:{}".format(event_data.body_as_str()))
+    except:
+        raise
+    finally:
+        await client.stop_async()
+
+Troubleshooting
+===============
+
+General
+-------
+
+The Event Hubs APIs generate exceptions that can fall into the following categories, along with the associated action you can take to try to fix them.
+
+- **User coding error:** System.ArgumentException, System.InvalidOperationException, System.OperationCanceledException, System.Runtime.Serialization.SerializationException. General action: try to fix the code before proceeding.
```

Comment:
This looks like it is copy/pasted from .NET?

Reply:
We haven't updated README. The original version of README has this .net stuffs. We'll fix this problem when we update README. You see this file changed because bot AutorestCI updated a couple of files including README.rst with a template's contents and comitted. I reverted that commit. That bot thing was also fixed with the help from engineering team.